geoh5py/__init__.py

```diff
@@ -24,6 +24,15 @@
 from . import groups, objects
 from .groups import CustomGroup
 
+try:
+    from ._version import __version__
+except ModuleNotFoundError:
+    raise (
+        ModuleNotFoundError(
+            "geoh5py is not correctly initialized. Please check if it is correctly built."
```

Comment:
"built", or more "installed"?

I do not expect a user to be able to check is it built

Reply:
The `_version.py` file is generated during build. This message is intended more for developers than for users. `_version.py` is automatically delivered, but is not tracked by git, since it is a generated file.
What do you think @domfournier?

Reply:
Correct. User would have it installed anyway. Only a developer would have a non-installed local checkout.
Still, it is more about installing with pip (or pip -e) than about building